sdk/src/main/java/io/dapr/client/domain/Job.java

```diff
@@ -0,0 +1,74 @@
+package io.dapr.client.domain;
+
+import com.google.protobuf.Message;
+
+/**
+ * A Job to schedule
+ *
+ * @param <T> The class type of Job data.
+ */
+public final class Job<T extends Message> {
```

Comment:
Same. Make it gRPC agnostic.

Reply:
@charan2628 there is no need to make it parameterizable (`<T extend Message>`) here.